opengrok-web/src/main/java/org/opengrok/web/WebappListener.java

```diff
@@ -116,21 +116,21 @@ public void contextInitialized(final ServletContextEvent servletContextEvent) {
      * is marked as not being indexed.
      * @param env runtime environment
      */
-    private void check_index(RuntimeEnvironment env) {
+    private void checkIndex(RuntimeEnvironment env) {
         if (env.isProjectsEnabled()) {
             LOGGER.log(Level.FINE, "Checking indexes for all projects");
             Map<String, Project> projects = env.getProjects();
             File indexRoot = new File(env.getDataRootPath(), IndexDatabase.INDEX_DIR);
             if (indexRoot.exists()) {
-                for (String projectName : projects.keySet()) {
+                projects.keySet().forEach(projectName -> {
```

Comment:
I think that it doesn't make sense rewriting all `enhanced-for` loops to `forEach`. 

https://stackoverflow.com/questions/16635398/java-8-iterable-foreach-vs-foreach-loop

Reply:
Thanks for the pointer. Actually I wanted to use entrySet() and get lured into using `forEach` by the Sonar plugin in IDEA.